src/components/Questions/Questions.css

```diff
@@ -0,0 +1,51 @@
+* {
+  background: #f9f9fa;
+  margin: 0;
+}
+
+.qustion__card {
+  border-radius: 12px;
+  box-shadow: 0px 2px 6px rgba(0, 0, 0, 0.25);
+}
+.card-header {
```

Comment:
leave one empty line between blocks. here and all

Reply:
fixed